griptape/tools/date_time/tool.py

```diff
@@ -17,3 +18,22 @@ def get_current_datetime(self, _: dict) -> BaseArtifact:
             return TextArtifact(str(current_datetime))
         except Exception as e:
             return ErrorArtifact(f"error getting current datetime: {e}")
+    
+    @activity(config={
+        "uses_default_memory": False,
+        "description": "Can be used to return a relative date and time.",
+        "schema": Schema({
+            Literal(
+                "date_string",
+                description="A string representing date and/or time in a recognizably valid format."
```

Comment:
I think this might be too ambiguous for the LLM as well. May be it should be something like:
> Relative date in English. For example, "now EST", "20 minutes ago", or "yesterday at 2pm"